src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskTypesControllerTest.java

```diff
@@ -221,7 +223,8 @@ void should_return_503_with_application_problem_response_when_camunda_api_is_not
         when(camundaServiceApi.getTaskTypesDmnTable(
             anyString(),
             anyString(),
-            anyString()
+            anyString(),
+            anyBoolean()
```

Comment:
Do we have DMN tests where multiple DMNs are returned?

Reply:
no we don't have it at the moment.

Reply:
Again this can be a follow up ... I don't think any team uses it currently but it will happen in future.